SwiftCompilerSources/Sources/Optimizer/FunctionPasses/ClosureSpecialization.swift

```diff
@@ -132,26 +131,24 @@ let autodiffClosureSpecialization = FunctionPass(name: "autodiff-closure-special
   repeat {
     // TODO: Names here are pretty misleading. We are looking for a place where
     // the pullback closure is created (so for `partial_apply` instruction).
-    var callSites = gatherCallSites(in: function, context)
```

Comment:
Can we rename `callSite` while here? And resolve `TODO`?

Reply:
Fixed in df4a5c02e2d05ebccf6aaedf6e5763a81b1f4cac. Please let me know if the new naming is OK